diff --git b/extras/Projucer/Source/Project/jucer_Project.h a/extras/Projucer/Source/Project/jucer_Project.h
index 1a67d03..e0da4d8 100644
--- b/extras/Projucer/Source/Project/jucer_Project.h
+++ a/extras/Projucer/Source/Project/jucer_Project.h
@@ -221,8 +221,8 @@ public:
     static String getPluginDefinesFilename()                    { return "JucePluginDefines.h"; }
     static String getJuceSourceHFilename()                      { return "JuceHeader.h"; }
     static String getJuceLV2DefinesFilename()                   { return "JuceLV2Defines.h"; }
-    static String getLV2FileWriterName()                        { return "juce_lv2_helper"; }
-    static String getVST3FileWriterName()                       { return "juce_vst3_helper"; }
+    static String getLV2FileWriterName()                        { return "juce70_lv2_helper"; }
+    static String getVST3FileWriterName()                       { return "juce70_vst3_helper"; }
 
     //==============================================================================
     template <class FileType>
diff --git b/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp a/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp
index ba1a2ab..f9c7986 100644
--- b/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp
+++ a/extras/Projucer/Source/Settings/jucer_StoredSettings.cpp
@@ -348,11 +348,11 @@ static String getFallbackPathForOS (const Identifier& key, DependencyPathOS os)
 {
     if (key == Ids::jucePath)
     {
-        return (os == TargetOS::windows ? "C:\\JUCE" : "~/JUCE");
+        return (os == TargetOS::windows ? "C:\\JUCE" : "/opt/kxstudio/include/JUCE-7.0.12");
     }
     else if (key == Ids::defaultJuceModulePath)
     {
-        return (os == TargetOS::windows ? "C:\\JUCE\\modules" : "~/JUCE/modules");
+        return (os == TargetOS::windows ? "C:\\JUCE\\modules" : "/opt/kxstudio/include/JUCE-7.0.12/modules");
     }
     else if (key == Ids::defaultUserModulePath)
     {
