--- ntk-static-0.0.0+git20130603.orig/ntk.pc.in
+++ ntk-static-0.0.0+git20130603/ntk.pc.in
@@ -9,5 +9,5 @@ Version: @VERSION@

 Requires: cairo >= 1.9.0
 Requires.private: x11 xft
-Libs: -L${libdir} -lntk
+Libs: -L${libdir} -lntk -lcairo -lgobject-2.0 -lglib-2.0 -lpixman-1 -lpng16 -lfontconfig -lXft -lfreetype -lXrender -lX11 -ldl -lz
 Cflags: -I${includedir}/ntk @CFLAGS@
--- ntk-static-0.0.0+git20130930.orig/ntk_images.pc.in
+++ ntk-static-0.0.0+git20130930/ntk_images.pc.in
@@ -9,5 +9,5 @@ Version: @VERSION@
 
 Requires: cairo >= 1.9.0
 Requires.private: x11 xft
-Libs: -L${libdir} -lntk_images -lntk
+Libs: -L${libdir} -lntk_images -lntk -lcairo -lgobject-2.0 -lglib-2.0 -lpixman-1 -lpng16 -lfontconfig -lXft -lfreetype -lXrender -lX11 -ldl -lz
 Cflags: -I${includedir}/ntk @CFLAGS@
