--- lsp-plugins-1.2.3.orig/make/tools.mk
+++ lsp-plugins-1.2.3/make/tools.mk
@@ -150,8 +150,8 @@ CXXDEFS            += -DLSP_INSTALL_PREF
 
 INCLUDE            :=
 LDFLAGS            := $(LDFLAGS_EXT) -r
-EXE_FLAGS          := $(EXE_FLAGS_EXT) $(FLAG_RELRO) -Wl,--gc-sections
-SO_FLAGS           := $(SO_FLAGS_EXT) $(FLAG_RELRO) -Wl,--gc-sections -shared $(FLAG_STDLIB) -fPIC 
+EXE_FLAGS          += $(EXE_FLAGS_EXT) $(FLAG_RELRO) -Wl,--gc-sections
+SO_FLAGS           += $(SO_FLAGS_EXT) $(FLAG_RELRO) -Wl,--gc-sections -shared $(FLAG_STDLIB) -fPIC 
 
 # Define flags for host build
 HOST_CFLAGS        := $(CFLAGS)
