diff --git a/extras/Build/CMake/JUCEConfig.cmake.in b/extras/Build/CMake/JUCEConfig.cmake.in
index 6a96e24e5..17b0911f4 100644
--- a/extras/Build/CMake/JUCEConfig.cmake.in
+++ b/extras/Build/CMake/JUCEConfig.cmake.in
@@ -22,10 +22,12 @@
 
 include("${CMAKE_CURRENT_LIST_DIR}/LV2_HELPER.cmake")
 
-if(NOT TARGET juce::juceaide)
-    add_executable(juce::juceaide IMPORTED)
-    set_target_properties(juce::juceaide PROPERTIES
-        IMPORTED_LOCATION "@PACKAGE_JUCEAIDE_PATH@")
+set(PACKAGE_PREFIX_DIR "@CMAKE_INSTALL_PREFIX@")
+
+if(NOT TARGET juceaide7)
+    add_executable(juceaide7 IMPORTED)
+    set_target_properties(juceaide7 PROPERTIES
+        IMPORTED_LOCATION "${PACKAGE_PREFIX_DIR}/bin/juceaide7")
 endif()
 
 check_required_components("@PROJECT_NAME@")
diff --git a/extras/Build/CMake/JUCEUtils.cmake b/extras/Build/CMake/JUCEUtils.cmake
index 58e65087c..33f90f750 100644
--- a/extras/Build/CMake/JUCEUtils.cmake
+++ b/extras/Build/CMake/JUCEUtils.cmake
@@ -192,7 +192,7 @@ function(_juce_add_au_resource_fork shared_code_target au_target)
     # In the end, it's simplest to generate a special single-purpose appconfig just for the
     # resource compiler.
     add_custom_command(OUTPUT "${secret_au_plugindefines}"
-        COMMAND juce::juceaide auplugindefines "${defs_file}" "${secret_au_plugindefines}"
+        COMMAND juceaide7 auplugindefines "${defs_file}" "${secret_au_plugindefines}"
         DEPENDS "${defs_file}"
         VERBATIM)
 
@@ -414,7 +414,7 @@ function(juce_add_binary_data target)
     file(WRITE "${input_file_list}" "${newline_delimited_input}")
 
     add_custom_command(OUTPUT ${binary_file_names}
-        COMMAND juce::juceaide binarydata "${JUCE_ARG_NAMESPACE}" "${JUCE_ARG_HEADER_NAME}"
+        COMMAND juceaide7 binarydata "${JUCE_ARG_NAMESPACE}" "${JUCE_ARG_HEADER_NAME}"
             ${juce_binary_data_folder} "${input_file_list}"
         WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
         DEPENDS "${input_file_list}"
@@ -498,7 +498,7 @@ function(juce_generate_juce_header target)
     set(extra_args)
 
     add_custom_command(OUTPUT "${juce_header}"
-        COMMAND juce::juceaide header "${defs_file}" "${juce_header}" ${extra_args}
+        COMMAND juceaide7 header "${defs_file}" "${juce_header}" ${extra_args}
         DEPENDS "${defs_file}"
         VERBATIM)
 endfunction()
@@ -506,23 +506,23 @@ endfunction()
 # ==================================================================================================
 
 function(_juce_execute_juceaide)
-    if(NOT TARGET juce::juceaide)
+    if(NOT TARGET juceaide7)
         message(FATAL_ERROR "The juceaide target does not exist")
     endif()
 
-    get_target_property(juceaide_location juce::juceaide IMPORTED_LOCATION)
+    get_target_property(juceaide7_location juceaide7 IMPORTED_LOCATION)
 
-    if(NOT EXISTS "${juceaide_location}")
-        message(FATAL_ERROR "juceaide was imported, but it doesn't exist!")
+    if(NOT EXISTS "${juceaide7_location}")
+        message(FATAL_ERROR "juceaide7 was imported, but it doesn't exist!")
     endif()
 
-    execute_process(COMMAND "${juceaide_location}" ${ARGN}
+    execute_process(COMMAND "${juceaide7_location}" ${ARGN}
         RESULT_VARIABLE result_variable
         OUTPUT_VARIABLE output
         ERROR_VARIABLE output)
 
     if(result_variable)
-        message(FATAL_ERROR "Running juceaide failed:\n${output}")
+        message(FATAL_ERROR "Running juceaide7 failed:\n${output}")
     endif()
 endfunction()
 
@@ -659,7 +659,7 @@ function(_juce_configure_bundle source_target dest_target)
         MACOSX_BUNDLE_INFO_PLIST "${this_output_plist}")
 
     add_custom_command(OUTPUT "${this_output_pkginfo}"
-        COMMAND juce::juceaide pkginfo "${juce_kind_string}" "${this_output_pkginfo}"
+        COMMAND juceaide7 pkginfo "${juce_kind_string}" "${this_output_pkginfo}"
         VERBATIM)
 
     set(output_folder "$<TARGET_BUNDLE_CONTENT_DIR:${dest_target}>")
@@ -751,7 +751,7 @@ function(_juce_add_resources_rc source_target dest_target)
     set(resource_rc_file "${juce_library_code}/resources.rc")
 
     add_custom_command(OUTPUT "${resource_rc_file}"
-        COMMAND juce::juceaide rcfile "${input_info_file}" "${resource_rc_file}"
+        COMMAND juceaide7 rcfile "${input_info_file}" "${resource_rc_file}"
         ${dependency}
         VERBATIM)
 
@@ -1024,7 +1024,7 @@ function(_juce_set_plugin_target_properties shared_code_target kind)
         set_target_properties(${target_name} PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${output_path}")
 
         add_custom_command(TARGET ${target_name} POST_BUILD
-            COMMAND juce::juce_lv2_helper "$<TARGET_FILE:${target_name}>"
+            COMMAND juce7_lv2_helper "$<TARGET_FILE:${target_name}>"
             VERBATIM)
 
         _juce_set_copy_properties(${shared_code_target} ${target_name} "${output_path}" JUCE_LV2_COPY_DIR)
