diff --git a/include/lilv/lilv.h b/include/lilv/lilv.h
index 5fd7a9f..368df82 100644
--- a/include/lilv/lilv.h
+++ b/include/lilv/lilv.h
@@ -27,15 +27,7 @@
 #include <stdint.h>
 #include <stdio.h>
 
-#if defined(_WIN32) && !defined(LILV_STATIC) && defined(LILV_INTERNAL)
-#  define LILV_API __declspec(dllexport)
-#elif defined(_WIN32) && !defined(LILV_STATIC)
-#  define LILV_API __declspec(dllimport)
-#elif defined(__GNUC__)
-#  define LILV_API __attribute__((visibility("default")))
-#else
-#  define LILV_API
-#endif
+#define LILV_API
 
 #if defined(__GNUC__) && \
   (__GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ >= 1))
