--- opus-static-1.3.1.orig/include/opus_defines.h
+++ opus-static-1.3.1/include/opus_defines.h
@@ -63,19 +63,7 @@ extern "C" {
 /** @cond OPUS_INTERNAL_DOC */
 /**Export control for opus functions */
 
-#ifndef OPUS_EXPORT
-# if defined(_WIN32)
-#  if defined(OPUS_BUILD) && defined(DLL_EXPORT)
-#   define OPUS_EXPORT __declspec(dllexport)
-#  else
-#   define OPUS_EXPORT
-#  endif
-# elif defined(__GNUC__) && defined(OPUS_BUILD)
-#  define OPUS_EXPORT __attribute__ ((visibility ("default")))
-# else
-#  define OPUS_EXPORT
-# endif
-#endif
+#define OPUS_EXPORT
 
 # if !defined(OPUS_GNUC_PREREQ)
 #  if defined(__GNUC__)&&defined(__GNUC_MINOR__)
