diff --git a/include/serd/serd.h b/include/serd/serd.h
index 2e7a13e..fb99b48 100644
--- a/include/serd/serd.h
+++ b/include/serd/serd.h
@@ -12,18 +12,7 @@
 #include <stdint.h>
 #include <stdio.h>
 
-// SERD_API must be used to decorate things in the public API
-#ifndef SERD_API
-#  if defined(_WIN32) && !defined(SERD_STATIC) && defined(SERD_INTERNAL)
-#    define SERD_API __declspec(dllexport)
-#  elif defined(_WIN32) && !defined(SERD_STATIC)
-#    define SERD_API __declspec(dllimport)
-#  elif defined(__GNUC__)
-#    define SERD_API __attribute__((visibility("default")))
-#  else
-#    define SERD_API
-#  endif
-#endif
+#define SERD_API
 
 #ifdef __GNUC__
 #  define SERD_PURE_FUNC __attribute__((pure))
