diff --git a/include/serd/serd.h b/include/serd/serd.h
index e550541..c2ee87d 100644
--- a/include/serd/serd.h
+++ b/include/serd/serd.h
@@ -25,15 +25,7 @@
 #include <stdint.h>
 #include <stdio.h>
 
-#if defined(_WIN32) && !defined(SERD_STATIC) && defined(SERD_INTERNAL)
-#  define SERD_API __declspec(dllexport)
-#elif defined(_WIN32) && !defined(SERD_STATIC)
-#  define SERD_API __declspec(dllimport)
-#elif defined(__GNUC__)
-#  define SERD_API __attribute__((visibility("default")))
-#else
-#  define SERD_API
-#endif
+#define SERD_API
 
 #ifdef __GNUC__
 #  define SERD_PURE_FUNC __attribute__((pure))
