--- fluidsynth-static-1.1.11.orig/include/fluidsynth.cmake
+++ fluidsynth-static-1.1.11/include/fluidsynth.cmake
@@ -29,28 +29,8 @@ extern "C" {
 
 #cmakedefine01 BUILD_SHARED_LIBS
 
-#if (BUILD_SHARED_LIBS == 0)
-    #define FLUIDSYNTH_API // building static lib? no visibility control then
-#elif defined(WIN32)
-    #if defined(FLUIDSYNTH_NOT_A_DLL)
-        #define FLUIDSYNTH_API
-    #elif defined(FLUIDSYNTH_DLL_EXPORTS)
-        #define FLUIDSYNTH_API __declspec(dllexport)
-    #else
-        #define FLUIDSYNTH_API __declspec(dllimport)
-    #endif
-
-#elif defined(MACOS9)
-#define FLUIDSYNTH_API __declspec(export)
-
-#elif defined(__GNUC__)
-#define FLUIDSYNTH_API __attribute__ ((visibility ("default")))
-
-#else
 #define FLUIDSYNTH_API
 
-#endif
-
 #if defined(__GNUC__) || defined(__clang__)
 #    define FLUID_DEPRECATED __attribute__((deprecated))
 #elif defined(_MSC_VER) && _MSC_VER > 1200
