diff --git a/source/frontend/carla_database.py b/source/frontend/carla_database.py
index 98b97bada..7654851e2 100755
--- a/source/frontend/carla_database.py
+++ b/source/frontend/carla_database.py
@@ -239,12 +239,12 @@ def runCarlaDiscovery(itype, stype, filename, tool, wineSettings=None):
         else:
             break
 
-        if line == "carla-discovery::init::-----------":
+        if line.startswith("carla-discovery::init::"):
             pinfo = deepcopy(PyPluginInfo)
             pinfo['type']     = itype
             pinfo['filename'] = filename if filename != ":all" else ""
 
-        elif line == "carla-discovery::end::------------":
+        elif line.startswith("carla-discovery::end::"):
             if pinfo is not None:
                 plugins.append(pinfo)
                 del pinfo
diff --git a/source/plugin/Makefile b/source/plugin/Makefile
index 24a4d4c28..19d9f973a 100644
--- a/source/plugin/Makefile
+++ b/source/plugin/Makefile
@@ -134,9 +134,17 @@ SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay16_plugin
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay32_plugin
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay64_plugin
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay_cv_plugin
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay_cv8_plugin
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay_cv32_plugin
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_patchbay_obs_plugin
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_create_native_plugin_host_handle
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_host_handle_free
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_native_plugin_engine
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_complete_license_text
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_juce_version
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_supported_file_extensions
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_supported_features
+SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_library_filename
 SYMBOLS_NATIVE += -Wl,-exported_symbol,_carla_get_library_folder
 SYMBOLS_LV2     = -Wl,-exported_symbol,_lv2_descriptor
 ifeq ($(HAVE_PYQT),true)
@@ -152,9 +160,18 @@ SYMBOLS_NATIVE += 'carla_get_native_patchbay16_plugin',
 SYMBOLS_NATIVE += 'carla_get_native_patchbay32_plugin',
 SYMBOLS_NATIVE += 'carla_get_native_patchbay64_plugin',
 SYMBOLS_NATIVE += 'carla_get_native_patchbay_cv_plugin',
+SYMBOLS_NATIVE += 'carla_get_native_patchbay_cv8_plugin',
+SYMBOLS_NATIVE += 'carla_get_native_patchbay_cv32_plugin',
+SYMBOLS_NATIVE += 'carla_get_native_patchbay_obs_plugin',
 SYMBOLS_NATIVE += 'carla_create_native_plugin_host_handle',
 SYMBOLS_NATIVE += 'carla_host_handle_free',
-SYMBOLS_NATIVE += 'carla_get_native_plugin_engine'
+SYMBOLS_NATIVE += 'carla_get_native_plugin_engine',
+SYMBOLS_NATIVE += 'carla_get_complete_license_text',
+SYMBOLS_NATIVE += 'carla_get_juce_version',
+SYMBOLS_NATIVE += 'carla_get_supported_file_extensions',
+SYMBOLS_NATIVE += 'carla_get_supported_features',
+SYMBOLS_NATIVE += 'carla_get_library_filename',
+SYMBOLS_NATIVE += 'carla_get_library_folder'
 SYMBOLS_NATIVE += ]"
 ifeq ($(HAVE_PYQT),true)
 SYMBOLS_LV2     = -sSIDE_MODULE=2 -sEXPORTED_FUNCTIONS="['lv2_descriptor','lv2ui_descriptor']"
diff --git a/source/plugin/carla-host-plugin.cpp b/source/plugin/carla-host-plugin.cpp
index 5bc748582..247efbf34 100644
--- a/source/plugin/carla-host-plugin.cpp
+++ b/source/plugin/carla-host-plugin.cpp
@@ -18,6 +18,7 @@
 #include "CarlaNativePlugin.h"
 
 #define CARLA_PLUGIN_BUILD
+#define CARLA_HOST_PLUGIN_BUILD
 
 #include "carla-native-plugin.cpp"
 
diff --git a/source/plugin/carla-native-plugin.cpp b/source/plugin/carla-native-plugin.cpp
index 0ea582d22..ce97b45a4 100644
--- a/source/plugin/carla-native-plugin.cpp
+++ b/source/plugin/carla-native-plugin.cpp
@@ -26,7 +26,7 @@
 // --------------------------------------------------------------------------------------------------------------------
 // Expose info functions as needed
 
-#ifndef CARLA_PLUGIN_BUILD
+#ifndef CARLA_HOST_PLUGIN_BUILD
 # include "utils/Information.cpp"
 #endif
 
diff --git a/source/plugin/symbols/carla-native-plugin.def b/source/plugin/symbols/carla-native-plugin.def
index 0e5445452..e222e9c2a 100644
--- a/source/plugin/symbols/carla-native-plugin.def
+++ b/source/plugin/symbols/carla-native-plugin.def
@@ -16,3 +16,6 @@ EXPORTS
     carla_get_supported_features @13
     carla_get_library_filename @14
     carla_get_library_folder @15
+    carla_get_native_patchbay_cv8_plugin @16
+    carla_get_native_patchbay_cv32_plugin @17
+    carla_get_native_patchbay_obs_plugin @18
diff --git a/source/plugin/symbols/carla-native-plugin.txt b/source/plugin/symbols/carla-native-plugin.txt
index 896cfa6fc..060bb5528 100644
--- a/source/plugin/symbols/carla-native-plugin.txt
+++ b/source/plugin/symbols/carla-native-plugin.txt
@@ -4,6 +4,9 @@ carla_get_native_patchbay16_plugin
 carla_get_native_patchbay32_plugin
 carla_get_native_patchbay64_plugin
 carla_get_native_patchbay_cv_plugin
+carla_get_native_patchbay_cv8_plugin
+carla_get_native_patchbay_cv32_plugin
+carla_get_native_patchbay_obs_plugin
 carla_create_native_plugin_host_handle
 carla_host_handle_free
 carla_get_native_plugin_engine
diff --git a/source/plugin/symbols/carla-native-plugin.version b/source/plugin/symbols/carla-native-plugin.version
index f6285765b..855b98410 100644
--- a/source/plugin/symbols/carla-native-plugin.version
+++ b/source/plugin/symbols/carla-native-plugin.version
@@ -5,6 +5,9 @@
             carla_get_native_patchbay32_plugin;
             carla_get_native_patchbay64_plugin;
             carla_get_native_patchbay_cv_plugin;
+            carla_get_native_patchbay_cv8_plugin;
+            carla_get_native_patchbay_cv32_plugin;
+            carla_get_native_patchbay_obs_plugin;
             carla_create_native_plugin_host_handle;
             carla_host_handle_free;
             carla_get_native_plugin_engine;
