--- dragonfly-reverb-3.2.6.orig/Makefile
+++ dragonfly-reverb-3.2.6/Makefile
@@ -45,6 +45,10 @@ clean:
 	$(MAKE) clean -C plugins/dragonfly-plate-reverb
 	$(MAKE) clean -C plugins/dragonfly-early-reflections
 	rm -rf bin build
+	rm -f common/*.d common/*.o
+	rm -f common/freeverb/*.d common/freeverb/*.o
+	rm -f common/kiss_fft/*.d common/kiss_fft/*.o
+	rm -f dpf/utils/lv2_ttl_generator.d
 
 # --------------------------------------------------------------
 
