From a51cb460bc958b60c7b40c0d27943a136fe1f49b Mon Sep 17 00:00:00 2001
From: falkTX <falktx@falktx.com>
Date: Sat, 15 Jul 2023 19:39:53 +0200
Subject: [PATCH] Fix previous commit

Signed-off-by: falkTX <falktx@falktx.com>
---
 plugins/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/Makefile b/plugins/Makefile
index c4d1a7b4..714bbce3 100644
--- a/plugins/Makefile
+++ b/plugins/Makefile
@@ -1405,7 +1405,7 @@ JACK_RESOURCES += $(CURDIR)/surgext/build/surge-data/fx_presets
 JACK_RESOURCES += $(CURDIR)/surgext/build/surge-data/wavetables
 JACK_RESOURCES += $(CURDIR)/surgext/build/surge-data/windows.wt
 
-RESOURCE_FILES += Cardinal/res/*.png
+RESOURCE_FILES += $(wildcard Cardinal/res/*.png)
 RESOURCE_FILES += Cardinal/res/Miku/Miku.png
 
 MINIPLUGIN_LIST     = Cardinal
