From ff22f1d357f278b1b3fd1eb2fd8354cd6c6a226a Mon Sep 17 00:00:00 2001
From: David Robillard <d@drobilla.net>
Date: Thu, 11 Mar 2021 13:33:31 -0500
Subject: [PATCH] Add missing class label

---
 schemas.lv2/dcs.ttl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/schemas.lv2/dcs.ttl b/schemas.lv2/dcs.ttl
index 37b1e109..44bf294e 100644
--- a/schemas.lv2/dcs.ttl
+++ b/schemas.lv2/dcs.ttl
@@ -180,7 +180,9 @@ dcs:file
 	rdfs:comment "A file involved in the change."@en ;
 	rdfs:domain dcs:Change .
 
-dbug:Issue a rdfs:Class .
+dbug:Issue
+	a rdfs:Class ;
+	rdfs:label "Issue"@en .
 
 dcs:fixes
 	a owl:ObjectProperty ;
