From e0b9a3c2806f846c104ba2e0c5816886ab310c1b Mon Sep 17 00:00:00 2001
From: David Robillard <d@drobilla.net>
Date: Thu, 11 Mar 2021 13:45:29 -0500
Subject: [PATCH] Sort xsd.ttl

---
 schemas.lv2/xsd.ttl | 36 ++++++++++++++++++------------------
 1 file changed, 18 insertions(+), 18 deletions(-)

diff --git a/schemas.lv2/xsd.ttl b/schemas.lv2/xsd.ttl
index 01be2e4d..47030624 100644
--- a/schemas.lv2/xsd.ttl
+++ b/schemas.lv2/xsd.ttl
@@ -7,9 +7,6 @@ xsd:
 	a owl:Ontology ;
 	rdfs:comment "XML Schema Datatypes" .
 
-xsd:anySimpleType
-	a rdfs:Datatype .
-
 xsd:ENTITY
 	a rdfs:Datatype ;
 	owl:onDatatype xsd:NCName .
@@ -44,6 +41,9 @@ xsd:QName
 		]
 	) .
 
+xsd:anySimpleType
+	a rdfs:Datatype .
+
 xsd:anyURI
 	a rdfs:Datatype ;
 	rdfs:label "URI reference" ;
@@ -352,21 +352,6 @@ xsd:pattern
 	rdfs:comment "A regular expression that matches complete valid literals." ;
 	rdfs:label "pattern" .
 
-xsd:whiteSpace
-	a rdf:Property ,
-		owl:DatatypeProperty ;
-	rdfs:comment "A string that describes whitespace normalization for a string type." ;
-	rdfs:label "white space" ;
-	rdfs:range [
-		a rdfs:Datatype ;
-		owl:onDatatype xsd:string ;
-		owl:withRestrictions (
-			[
-				xsd:pattern "(preserve|replace|collapse)"
-			]
-		)
-	] .
-
 xsd:positiveInteger
 	a rdfs:Datatype ;
 	rdfs:label "positive integer" ;
@@ -472,3 +457,18 @@ xsd:unsignedShort
 		]
 	) .
 
+xsd:whiteSpace
+	a rdf:Property ,
+		owl:DatatypeProperty ;
+	rdfs:comment "A string that describes whitespace normalization for a string type." ;
+	rdfs:label "white space" ;
+	rdfs:range [
+		a rdfs:Datatype ;
+		owl:onDatatype xsd:string ;
+		owl:withRestrictions (
+			[
+				xsd:pattern "(preserve|replace|collapse)"
+			]
+		)
+	] .
+
