From 7f7451290ce42c2542b427ea0402930e5af7d899 Mon Sep 17 00:00:00 2001
From: David Robillard <d@drobilla.net>
Date: Sat, 13 Mar 2021 12:44:44 -0500
Subject: [PATCH] Remove old link to SVN repository

---
 lv2/core/meta.ttl | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lv2/core/meta.ttl b/lv2/core/meta.ttl
index b626a0c9..5d646e58 100644
--- a/lv2/core/meta.ttl
+++ b/lv2/core/meta.ttl
@@ -25,10 +25,6 @@ THIS SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH R
 	doap:homepage <http://lv2plug.in/> ;
 	doap:mailing-list <http://lists.lv2plug.in/listinfo.cgi/devel-lv2plug.in> ;
 	doap:programming-language "C" ;
-	doap:repository [
-		a doap:SVNRepository ;
-		doap:location <http://lv2plug.in/repo>
-	] ;
 	doap:developer <http://drobilla.net/drobilla#me> ,
 		<http://plugin.org.uk/swh.xrdf#me> ;
 	doap:helper meta:larsl ,
