From 6820b96062ac5942fe797a6bae84f22a92e18b45 Mon Sep 17 00:00:00 2001
From: falkTX <falktx@falktx.com>
Date: Mon, 15 Mar 2021 12:15:06 +0000
Subject: [PATCH] Make sure to link against libsamplerate

Signed-off-by: falkTX <falktx@falktx.com>
---
 plugins/eg-sampler.lv2/wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/eg-sampler.lv2/wscript b/plugins/eg-sampler.lv2/wscript
index 4d9debc6..115d8a00 100644
--- a/plugins/eg-sampler.lv2/wscript
+++ b/plugins/eg-sampler.lv2/wscript
@@ -53,7 +53,7 @@ def build(bld):
               name         = 'sampler',
               target       = 'lv2/%s/sampler' % bundle,
               install_path = '${LV2DIR}/%s' % bundle,
-              use          = ['M', 'SNDFILE', 'LV2'])
+              use          = ['M', 'SNDFILE', 'SAMPLERATE', 'LV2'])
 
     # Build UI library
     if bld.env.HAVE_GTK2:
