--- suil-static-0.10.12.orig/include/suil/suil.h
+++ suil-static-0.10.12/include/suil/suil.h
@@ -24,13 +24,8 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#ifdef _WIN32
-#  define SUIL_LIB_IMPORT __declspec(dllimport)
-#  define SUIL_LIB_EXPORT __declspec(dllexport)
-#else
-#  define SUIL_LIB_IMPORT __attribute__((visibility("default")))
-#  define SUIL_LIB_EXPORT __attribute__((visibility("default")))
-#endif
+#define SUIL_LIB_IMPORT
+#define SUIL_LIB_EXPORT __attribute__((visibility("default")))
 
 #ifndef SUIL_STATIC
 #  ifdef SUIL_INTERNAL
