--- suil-static-0.10.12.orig/src/suil_internal.h
+++ suil-static-0.10.12/src/suil_internal.h
@@ -113,8 +113,12 @@ suil_host_init(void);
 {
 #define N_SLICES 4
 
+#ifdef KXSTUDIO_SUIL_MODULE_DIR
+  const char* const mod_dir = KXSTUDIO_SUIL_MODULE_DIR;
+#else
   const char* const env_dir = getenv("SUIL_MODULE_DIR");
   const char* const mod_dir = env_dir ? env_dir : SUIL_MODULE_DIR;
+#endif
 
   const char* const slices[N_SLICES] = {
     mod_dir, SUIL_DIR_SEP SUIL_MODULE_PREFIX, module_name, SUIL_MODULE_EXT};
