--- suil-static-0.10.12.orig/src/suil_internal.h
+++ suil-static-0.10.12/src/suil_internal.h
@@ -113,8 +113,12 @@ suil_host_init(void);
 static inline void*
 suil_open_module(const char* module_name)
 {
+#ifdef KXSTUDIO_SUIL_MODULE_DIR
+  const char* const mod_dir = KXSTUDIO_SUIL_MODULE_DIR;
+#else
   const char* const env_dir = getenv("SUIL_MODULE_DIR");
   const char* const mod_dir = env_dir ? env_dir : SUIL_MODULE_DIR;
+#endif
   const size_t      path_len =
     strlen(mod_dir) + strlen(SUIL_DIR_SEP SUIL_MODULE_PREFIX SUIL_MODULE_EXT) +
     strlen(module_name) + 2;
