diff --git a/include/FLAC/export.h b/include/FLAC/export.h
index cde72b7..3a60cf0 100644
--- a/include/FLAC/export.h
+++ b/include/FLAC/export.h
@@ -61,26 +61,8 @@
  * build static, shared or **both**. Therefore, DLL_EXPORT, which is set
  * by libtool, must override FLAC__NO_DLL on building shared components
  */
-#if defined(_WIN32)
-
-#if defined(FLAC__NO_DLL) && !(defined(DLL_EXPORT))
-#define FLAC_API
-#else
-#ifdef FLAC_API_EXPORTS
-#define	FLAC_API __declspec(dllexport)
-#else
-#define FLAC_API __declspec(dllimport)
-#endif
-#endif
-
-#elif defined(FLAC__USE_VISIBILITY_ATTR)
-#define FLAC_API __attribute__ ((visibility ("default")))
-
-#else
 #define FLAC_API
 
-#endif
-
 /** These \#defines will mirror the libtool-based library version number, see
  * http://www.gnu.org/software/libtool/manual/libtool.html#Libtool-versioning
  */
diff --git a/include/FLAC++/export.h b/include/FLAC++/export.h
index 19370fa..45b4752 100644
--- a/include/FLAC++/export.h
+++ b/include/FLAC++/export.h
@@ -72,21 +72,7 @@
  * build static, shared or **both**. Therefore, DLL_EXPORT, which is set
  * by libtool, must override FLAC__NO_DLL on building shared components
  */
-#if defined(_WIN32)
-#if defined(FLAC__NO_DLL) && !(defined(DLL_EXPORT))
 #define FLACPP_API
-#else
-#ifdef FLACPP_API_EXPORTS
-#define	FLACPP_API __declspec(dllexport)
-#else
-#define FLACPP_API __declspec(dllimport)
-#endif
-#endif
-#elif defined(FLAC__USE_VISIBILITY_ATTR)
-#define FLACPP_API __attribute__ ((visibility ("default")))
-#else
-#define FLACPP_API
-#endif
 
 /** These \#defines will mirror the libtool-based library version number, see
  * http://www.gnu.org/software/libtool/manual/libtool.html#Libtool-versioning
