diff --git a/include/sratom/sratom.h b/include/sratom/sratom.h
index 26966ee..61f2e20 100644
--- a/include/sratom/sratom.h
+++ b/include/sratom/sratom.h
@@ -30,15 +30,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#if defined(_WIN32) && !defined(SRATOM_STATIC) && defined(SRATOM_INTERNAL)
-#  define SRATOM_API __declspec(dllexport)
-#elif defined(_WIN32) && !defined(SRATOM_STATIC)
-#  define SRATOM_API __declspec(dllimport)
-#elif defined(__GNUC__)
-#  define SRATOM_API __attribute__((visibility("default")))
-#else
-#  define SRATOM_API
-#endif
+#define SRATOM_API
 
 #ifdef __cplusplus
 extern "C" {
